redux/types.ts

```diff
@@ -1,9 +1,26 @@
 export interface State {
```

Comment:
This file is the best explanation of the new state structure --- and a great argument for typescript with redux